saleor/graphql/meta/tests/utils.py

```diff
@@ -0,0 +1,406 @@
+from ...tests.utils import get_graphql_content
+
+PRIVATE_KEY = "private_key"
+PRIVATE_VALUE = "private_vale"
+
+PUBLIC_KEY = "key"
+PUBLIC_KEY2 = "key2"
+PUBLIC_VALUE = "value"
+PUBLIC_VALUE2 = "value2"
+
+
+def item_contains_proper_public_metadata(
+    item_from_response,
+    item,
+    item_id,
+    key=PUBLIC_KEY,
+    value=PUBLIC_VALUE,
+):
+    if item_from_response["id"] != item_id:
+        return False
+    item.refresh_from_db()
```

Comment:
This looks like a side effect. I do not think it should be a part of this function.